x-pack/plugins/session_view/common/ecs/rule/index.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export interface RuleEcs {
```

Comment:
nit -> maybe it is time to update this file `packages/kbn-logging/src/ecs/rule.ts` and just use @kbn-logging for ecs

Reply:
Thanks for pointing this out @XavierM, I ended up removing it for now since we are not adding any value with this type on this PR.

But, for future PRs, we are going to use the `RuleEcs` type, I just took a look around and it seems to be declared similarly by other plugins:

<details>
<summary>x-pack/plugins/timelines/common/ecs/rule/index.ts</summary>

![code](https://user-images.githubusercontent.com/19270322/152863162-ee349b3e-4296-4eaf-aea5-2ae4d82f6cad.png)
</details>

<details>
<summary>x-pack/plugins/osquery/common/ecs/rule/index.ts</summary>

![code](https://user-images.githubusercontent.com/19270322/152863254-19a2e9c8-0dfc-40f2-972c-e9c5b97646ce.png)
</details>

<details>
<summary>x-pack/plugins/security_solution/common/ecs/rule/index.ts</summary>

![code](https://user-images.githubusercontent.com/19270322/152863324-81c71746-1f2b-444a-a8df-9cc5b0a2973d.png)
</details>



That raised some questions, In that case, should let `@kbn-logging` with a single type with all the combined fields and update that on the [docs](https://www.elastic.co/guide/en/ecs/master/ecs-rule.html) as well, or just update it to be the common fields across the plugins, and let other rules extends the interface from `@kbn-logging`?

Also, wondering if `@kbn-logging`  would be the correct place since it has a note on README saying: 
_"this package currently only contains logging types. The only concrete implementation
is still in `core` for now."_

So, It is like `@kbn-logging` types are meant only for logging types, and it seems the types use by the plugins have other purposes.